src/client/index.html

```diff
@@ -13,9 +13,40 @@
 		*:after {
 			box-sizing: inherit;
 		}
+
```

Comment:
Setup your vscode to use prettier to format HTML files on-save too

hurts my eyes to see these huge tabs instead of spaces LOL

Reply:
OK. Got Prettier to work on HTML files now! 💯